hnn_core/gui/gui.py

```diff
@@ -1048,21 +1048,32 @@ def _get_cell_specific_widgets(layout, style, location, data=None):
     return widgets_list, widgets_dict
 
 
+def _cell_spec_change(change, widget):
+    if change['new']:
+        widget.disabled = True
+    else:
+        widget.disabled = False
+
+
 def _get_rhythmic_widget(name, tstop_widget, layout, style, location,
-                         data=None, default_weights_ampa=None,
-                         default_weights_nmda=None, default_delays=None,
-                         sync_evinput=False):
+                         data={}, weights_ampa=None,
+                         weights_nmda=None, delays=None,
+                         n_drive_cells=None, cell_specific=None
+                         ):
     default_data = {
         'tstart': 0.,
         'tstart_std': 0.,
         'tstop': tstop_widget.value,
         'burst_rate': 7.5,
         'burst_std': 0,
-        'repeats': 1,
```

Comment:
any reason to remove `repeats`?

Reply:
I changed this to 'numspikes' to match the api for drive creation.